Server/sprawl_calculator/wup_calculator/data_definitions.py

```diff
@@ -5,3 +5,23 @@
 class WUPCalculationRequestActivating:
 	status: str
 	processing_id: str
+
+@dataclass
+class WUPCalculationRequestProcessing:
+	status: str
+	processing_id: str
```

Comment:
In addition, I just added created at and updated at to the documentation: https://github.com/geodesignhub/urban-sprawl-calculation-service/blob/master/api/sprawl-calculation-service.yaml#L286 , these should be added to the definitions as well.

Reply:
Ok, I will check and update same and pr for review.

Reply:
New fields are added to the dataclasses.